Code/BuildSystem/ProbingSrc/ArchitectureDetect.c

```diff
@@ -0,0 +1,38 @@
+#if defined(__clang__) || defined(__GNUC__)
+	#if defined(__clang__)
+		#warning "COMPILER:'clang'"

```

Comment:
Why is this a warning and the rest are messages?

Reply:
I thought that #pragma message is not supported on clang and gcc. But it is, so I'm going to use that for everything.